src/main/resources/application.yml

```diff
@@ -20,13 +20,18 @@ micronaut:
         mapping: "/swagger/**"
   security:
     enabled: true
```

Comment:
Maybe we need to disabled security by default, since there is not `secret` defined and add some doc on readme about that (enabled security & add a secret)

Reply:
Yes, that's a good idea i guess